client/src/styles/Card.css

```diff
@@ -0,0 +1,129 @@
+.card-parent-container {
+  height: 50vmin;
+  max-height: 50vmin;
+  width: 35vmin;
+  max-width: 35vmin;
+}
+
+.card-container {
+  display: grid;
+  grid-template-rows: 1fr 4fr 1fr 2fr 1fr;
+  grid-gap: 10;
+  height: 100%;
+  width: 100%;
+  border: 1vmin solid black;
+  border-radius: 2vmin;
+  padding: 1vmin 1vmin 1vmin 1vmin;
+}
+
+.name-mana-container {
+  display: grid;
+  grid-row-start: 1;
+  grid-row-end: 2;
+  overflow: hidden;
+  object-fit: cover;
+}
+
+.name {
```

Comment:
Maybe the card name could be a little bigger and bolder than the rest of the text on the card, like `font-size: 2vmin` and `font-weight: bold`